src/macros.rs

```diff
@@ -1,10 +1,305 @@
+//! Extra macro documentation.
+
 #[doc(hidden)]
 pub use fizyr_rpc_macros::interface as interface_impl;
 
 #[macro_export]
 /// Define an RPC interface.
+///
+/// This macro generates a module with a `Client` and `Server` struct,
+/// and some more helper types.
+///
+/// The client struct can be used to initiate requests and send stream messages.
```

Comment:
```suggestion
/// The client struct can be used to initiate requests and stream messages.
```
Or is this conceptually very different?

Reply:
Yes, it's different, you don't initiate stream messages, you just send them and then you're done :)